src/server/services/cms-content.ts

```diff
@@ -245,9 +246,28 @@ async function getFooter(requestID: requestID, forceRenew: boolean = false) {
     });
 }
 
-export async function fetchCMSCONTENT(
+function modifyFooterContent(content: CMSFooterContent) {
+  let blocks = content.blocks.map((block) => {
+    return {
+      ...block,
+      description: block.description ? parse(block.description) : null,
```

Comment:
Ik wilde eigenlijk een object maken voor footer right en footer left. Er zijn alleen een paar situaties waardoor ik denk dat het beter is de ast te sturen en in de fe netjes te plaatsen. 
Zo zijn er bijvoorbeeld twee maal p tags met een child erin. Bijvoorbeeld een stuk tekst waar een link in staat waarna de tekst weer doorgaat na de link. Dit zou een weinig fraai object opleveren dat minder sense maakt dan de ast IMO.